libs/ts-rest/nest/src/lib/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const TsRestAppRouteMetadataKey = Symbol('ts-rest-app-route');
+export const TsRestAppRouterMetadataKey = Symbol('ts-rest-app-router');
```

Comment:
I think rather than introducing a new metadata key, we can re-use `TsRestAppRouteMetadataKey` and we can easily determine whether it's a multi handler or single handler by checking the value using `isAppRoute`